src/components/CardAddress/ModalAddressCard/ModalAddressCardAction/style.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components'
+
+const ModalAddressCardStyled = styled.button`
```

Comment:
ModalAddressCardActionStyled. Always try to match the component names to the folder name.